uint32 id
float32 field_of_view
float32 min_range
float32 max_range
float32 range